README.md

```diff
@@ -186,7 +186,86 @@ homeserver:
   ...
     # Any other field to be copied in homeserver.yaml.
 
-# --- Docker configuration
+appservices:
+  # Optional. Define Matrix Application Services to be connected
+  # with your homeserver. For more details on AppServices, see
+  # https://spec.matrix.org/v1.2/application-service-api .
+  host:
+    # Optional. Application Services whose process is running on
+    # the host.
+    name:
```

Comment:
I think there's missing `-` here for `name:`, the rest will need indenting accordingly